recipes/pymde/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pymde" %}
+{% set version = "0.1.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pymde-{{ version }}.tar.gz
+  sha256: 202d50ed6f41911518054215b8423a85d527ed6c2b72a9c2fa37d3bbecf345f9
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
Since this is a compiled package, please add `skip: true  # [win]` to skip the build on Windows due to the missing dependency.